### ISSUE TITLE
res.emote doesn't appear to work

### ISSUE THREAD

**Issue by lwcooper** [flowdock/hubot-flowdock#80]
Can you make hubot emote a message? For example:

This sends as a normal message:

```
robot.respond /pie/i, (res) ->
    res.emote "I like pie"
```
Outputs `/me I like pie`, instead of the blue bar with just 'I like pie'
```
robot.respond /pie/i, (res) ->
    res.send "/me I like pie"
```

**Comment by Mumakil** [comment]
Yeah, it seems that by default the emote command is just an alias for send. We'll need to override the emote command and post an `action` message with the specified content instead of a regular `message`.

**Comment by lwcooper** [comment]
Ill give that a try and submit a PR if I get it working

**Comment by csMACnz** [comment]
Anyone figured this out yet? I am looking to implement this in another app using the api and can't seem to find any info.
